ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -256,7 +280,88 @@ + (BOOL)verifyHashForDiffUpdate:(NSString *)finalUpdateFolder
         return NO;
     }
     
+    CPLog(@"Expected hash: %@, actual hash: %@", expectedHash, updateContentsManifestHash);
+    
     return [updateContentsManifestHash isEqualToString:expectedHash];
 }
 
-@end
\ No newline at end of file
++ (NSString *)cleanPublicKey:(NSString *)publicKeyString
```

Comment:
From my point of view `Clean` is not a good name here, probably `preparePublicKeyDecoding` or something else could fit better

Reply:
@max-mironov I'll fix it, thx!